.evergreen/config.yml

```diff
@@ -1770,8 +1772,12 @@ tasks:
       commands:
         - func: "bootstrap mongo-orchestration"
           vars:
+            VERSION: "latest"
```

Comment:
VERSION should not be here. The VERSION gets set in the matrix here:
```
- matrix_name: "load-balancer"
  matrix_spec:
    platform: ubuntu-18.04
    mongodb-version: ["5.0", "latest"]   <---------- VERSION set here
    auth-ssl: "*"
 ...
```

We'll need to fix this in master as well.

Reply:
We also need to change "5.0" to "rapid". We can no longer test load-balancer support against 5.0. The only reason the test pass is because the above bug makes it so that the "5.0" test are actually using "latest".

Reply:
Done.